e2e/tests/ui-driven/src/helpers/context.ts

```diff
@@ -91,12 +94,27 @@ export async function setUpTestContext(
   return context;
 }
 
+export const externalPortalServiceProps = {
+  name: "An External Portal Service",
+  slug: "an-external-portal-service",
+};
+
+export const externalPortalFlowData = {
+  title: "Is this an External Portal?",
+  answers: ["It is an external portal", "No it is not an External Portal"],
+};
+
 export async function tearDownTestContext(context: Context) {
   const adminGQLClient = getGraphQLClient();
-  if (context.flow) {
+  if (context.flows?.[0]) {
     await deleteSession(adminGQLClient, context);
-    await deletePublishedFlow(adminGQLClient, context);
-    await deleteFlow(adminGQLClient, context);
+
+    for (const flow of context.flows) {
+      await deleteFlow(adminGQLClient, flow);
+      if (flow.publishedId) {
+        await deletePublishedFlow(adminGQLClient, flow);
+      }
+    }
```

Comment:
Have a look at using `$admin.flow._destroyAll()` here - I also think that published flows should be deleted via cascade when the source flow is deleted.

Reply:
There should be a good comparable example in the API e2e tests btw 👌